webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityStreamToolbar.vue

```diff
@@ -23,23 +23,46 @@
     height="52"
     flat
     dense>
-    <v-flex class="d-flex">
-      <div v-if="userCanPost" class="openLink my-auto pa-0 font-weight-bold text-truncate">
-        <a @click="openComposerDrawer(true)" class="hover-underline primary--text">
-          <i class="uiIconEdit me-1"></i>{{ composerButtonLabel }}
-        </a>
-      </div>
-      <div v-else>
-        <v-card-text class="text-sub-title text-uppercase center px-0">
-          {{ $t('activity.toolbar.title') }}
-        </v-card-text>
-      </div>
-      <div
-        v-if="streamFilterEnabled"
-        class="ms-auto my-auto">
-        <activity-stream-filter />
-      </div>
-    </v-flex>
+    <v-row class="d-flex flex-nowrap" no-gutters>
+      <v-col 
+        cols="9" 
+        lg="8" 
+        md="6" 
+        xl="9">
```

Comment:
Can you please use a simple `div` with classes `d-flex` & `flex-grow-1` & `flex-grow-0` &  & `flex-shrink-1`
 &  & `flex-shrink-0` for a better UX in all screens.

```suggestion
```

Reply:
done!